core/src/main/resources/schema/app/20201103154306_CB-9366_Create_load_balancer_metadata_table.sql

```diff
@@ -0,0 +1,62 @@
+-- // CB-9366 Create load balancer metadata table
+-- Migration SQL that makes the change goes here.
+
+CREATE SEQUENCE loadbalancer_id_seq START WITH 1
```

Comment:
Just would like to add some notes about the unwritten convention that is mostly used by our team in case of SQL scripts, this SQL script is fine for me.

You could save the time of sequence creator SQLs with using name convention and `bigserial` as type of id. In this case the `DROP TABLE` should also cleanup the sequence too. For example: 
 - [credential table](https://github.com/hortonworks/cloudbreak/blob/master/environment/src/main/resources/schema/app/20190510134440_CB-1210-create-credential-table.sql#L5)
 - [credential id sequence name](https://github.com/hortonworks/cloudbreak/blob/master/environment/src/main/java/com/sequenceiq/environment/credential/domain/Credential.java#L28)

On the other hand we prefer SQL statements with `IF EXISTS` or `IF NOT EXISTS` conditions in place where it is applicable, that makes the SQL scripts more bullet proof.

Reply:
I've revamped the SQL script using your example, and added `IF EXISTS` or `IF NOT EXISTS` checks to everything. Let me know if it looks okay.